uber/site_sections/summary.py

```diff
@@ -175,12 +175,101 @@ def food_eligible(self, session):
+                        # otherwise, make a new one and add it to the session for when we commit
+                        model_instance = model()
+                        session.add(model_instance)
+
+                for colname, val in row.items():
+                    col = cols[colname]
+                    if not val:
+                        # in a lot of cases we'll just have the empty string, so we'll just
+                        # do nothing for those cases
+                        continue
+                    if isinstance(col.type, Choice):
+                        # the export has labels, and we want to convert those back into their
+                        # integer values, so let's look that up (note: we could theoretically
+                        # modify the Choice class to do this automatically in the future)
+                        label_lookup = {val: key for key, val in col.type.choices.items()}
+                        val = label_lookup[val]
+                    elif isinstance(col.type, MultiChoice):
+                        # the export has labels separated by ' / ' and we want to convert that
+                        # back into a comma-separate list of integers
+                        label_lookup = {val: key for key, val in col.type.choices}
+                        vals = [label_lookup[label] for label in val.split(' / ')]
+                        val = ','.join(map(str, vals))
+                    elif isinstance(col.type, UTCDateTime):
+                        # we'll need to make sure we use whatever format string we used to
+                        # export this date in the first place
+                        try:
+                            val = UTC.localize(datetime.strptime(val, date_format + ' %H:%M:%S'))
+                        except:
+                            val = UTC.localize(datetime.strptime(val, date_format))
+                    elif isinstance(col.type, Date):
+                        val = datetime.strptime(val, date_format).date()
+                    elif isinstance(col.type, Integer):
+                        val = int(val)
+
+                    # now that we've converted val to whatever it actually needs to be, we
+                    # can just set it on the attendee
+                    setattr(model_instance, colname, val)
+
+                try:
+                    session.commit()
+                except:
+                    log.error('ImportError', exc_info=True)
+                    session.rollback()
+                    message = 'Import unsuccessful'
+
+                id_list.append(model_instance.id)
+
+            if id_list and model == Attendee:
+                all_instances = session.query(model).filter(model.id.in_(id_list)).all()
+
+        return {
+            'message': message,
+            'tables': sorted([model.__name__.lower() for model in Session.all_models()]),
+            'attendees': all_instances
+        }
+
+    def valid_attendees(self):
```

Comment:
We keep changing the URL for this, so I figured I'd be nice and leave in a redirect for our poor beleaguered admins.